lib/standard/kernel.nit

```diff
@@ -226,6 +226,17 @@ interface Discrete
 	end
 end
 
+# Something that can be cloned
+#
+# The main method on this interface is `clone` and the specific
```

Comment:
Suggestion:

~~~nit
# Something that can be cloned.
# 
# This interface introduces the `clone` method used to duplicate an instance.
interface Clonable
    # Duplicate `self`.
    # 
    # The specific semantic of this method is let to the subclasses.
    # For example you can choose that the cloning is done in depth by cloning
    # the linked instances or only shallow.
    #
    # See `Array::clone` for a shallow implementation example.
    # See `??` for an in depth clone example.
    #
    # Note: as the returned clone depends on the semantic, one should also want to redefine the
    # `==` method to specify the equality between an object and his clone.
    fun clone: SELF is abstract
end
~~~